content/scalers/apache-kafka-topic.md

```diff
@@ -19,7 +19,7 @@ This specification describes the `kafka` trigger for Apache Kafka Topic.
   triggers:
   - type: kafka
     metadata:
-      brokerList: kafka.svc:9092
```

Comment:
This is still supported so might want to keep it and mention it is deprecated?

Reply:
well, in this case, we should split the example into two different examples because you cannot have both fields together, or leaving with one example and commenting the `brokerList` line with a "deprecated".
Tbh, I would not have this in the doc anymore so that people can move to the new field.

Reply:
If it's documented in the spec above that's good enough.

We should still document the old approach as long as we support it but clearly mention that it is deprecated.

If we don't do that, existing users will be confused.

Reply:
So I would add a commented line in the example instead of having two different ones. I will update the PR.